app/src/interfaces/input/input.vue

```diff
@@ -78,6 +80,7 @@ const inputType = computed(() => {
 		:max-length="length"
 		:step="step"
 		:dir="direction"
+		:integer="isInteger"
```

Comment:
Actually, I wonder if this should `allowFloat` instead in order to be more explicit about what it's trying to signal? 🤔